CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/BlazeContextMenu.java

```diff
@@ -168,7 +169,7 @@ public void selectItem(final String item, final Graph graph, final GraphElementT
         if (elementType == GraphElementType.VERTEX) {
             return Arrays.asList(ADD_BLUE_BLAZE, ADD_RED_BLAZE, ADD_YELLOW_BLAZE, ADD_CUSTOM_BLAZE, UNSET_BLAZE);
         } else {
-            return Arrays.asList();
+            return Collections.EMPTY_LIST;
```

Comment:
use `Collections.emptyList()` instead of `Collections.EMPTY_LIST` (the former creates a typed List, the latter creates a raw List)

Reply:
Cheers, It didn't even click that it could have been different. I changed them all.